Resources/Maps/gaxstation.yml

```diff
@@ -128949,6 +129717,17 @@ entities:
     - type: Transform
       pos: -15.405928,70.28039
       parent: 2
+- proto: WeaponShotgunDoubleBarreledRubber
+  entities:
+  - uid: 20352
+    components:
+    - type: Transform
+      parent: 20345
+    - type: Physics
+      angularDamping: 0
+      linearDamping: 0
+      canCollide: False
+    - type: InsideEntityStorage
```

Comment:
```suggestion
```
Actually this should not be mapped. The Bartender gets their shotgun from their loadouts. They also have weapon options, in the form of optionally starting with a Mosin(rubber) in place of a shotgun.

Reply:
Oh, I didn't realize that, sorry